.github/ISSUE_TEMPLATE/gc_member_onboarding.md

```diff
@@ -0,0 +1,200 @@
+- [ ] Add new member as code owner in [open-telemetry/community](https://github.com/open-telemetry/community/blob/main/CODEOWNERS)
+- [ ] Add new member to the list of GC members in [open-telemetry/community/community-members.md](https://github.com/open-telemetry/community/blob/main/community-members.md).
+- [ ] Add new member in the `OpenTelemetry (Governance Committee)` section of [cncf/foundation/project-maintainers.csv](https://github.com/cncf/foundation/blob/main/project-maintainers.csv)
+
+### Mailing Lists
+The following mailing lists ensure that communications reach all necessary
+members of this group.
+
+The existing member must:
+
+- [ ] Invite new member as `Moderator`, ensuring that they have the same
+  privileges as existing members, to the following mailing lists:
+  - https://lists.cncf.io/g/cncf-opentelemetry-governance
+  - https://lists.cncf.io/g/cncf-otel-zoomadmins
+- [ ] Open CNCF Service Desk ticket to invite new member as `Member` to 
+  https://lists.cncf.io/g/cncf-opentelemetry-maintainers
+
+The new member must:
+- [ ] Accept invitations to mailing lists above, creating an account on https://lists.cncf.io
+  if necessary, using their preferred email address.
+
+### Google Drive
+OpenTelemetry handles minutes and other ad-hoc documents in Google Drive.
+
+The existing member must:
+
+- [ ] Give new member write access to:
+  - [OpenTelemetry Governance Committee meeting notes](https://docs.google.com/document/d/1-23Sf7-xZK3OL5Ogv2pK0NP9YotlSa0PKU9bvvtQwp8)
+
+The new member must:
+
+- [ ] Validate they have write access to the documents listed above.
+
+### Google Calendar
+All OpenTelemetry public meetings are handled via Google Calendars (see [How to handle public calendar](https://github.com/open-telemetry/community/blob/main/docs/how-to-handle-public-calendar.md)
+for more details). People in the  `opentelemetry-calendar-contributors` Google
+Group have write access to the OpenTelemetry public calendar. To view and manage
+members of this group one must use the `opentelemetry.calendar.owner@gmail.com`
+account.
+
+The existing member must:
+
+- [ ] Add user as `Owner` of https://groups.google.com/g/opentelemetry-calendar-contributors.
+
+The new member must:
+
+- [ ] Validate they have `Owner` permissions on the group above.
+
+### Zapier
+Zapier is used to sync meeting recordings from Zoom to a publicly viewable
+Google spreadsheet (see https://github.com/open-telemetry/community/blob/main/docs/how-meeting-recordings-upload-works.md
+for more details).
+
+The new member must:
+- [ ] Validate they can log in to https://zapier.com with the credentials in 1Password.
+
+## Meetings and Ceremonies
+As per GC charter, all members are expected to attend the following meetings.
+
+- Governance Committee Meeting (every Thursday 11:00 Pacific Time, which is
+  typically 19:00 UTC).
+- Governance/Technical Committee Joint Meeting (every second Thursday of the month, coinciding 
+  with the Governance Committee Meeting).
```

Comment:
Maybe worth to add a note that this time may be updated after each election cycle. Otherwise this may scare people off who can not make it possible to attend at that time.

Reply:
Good point. I'll add a note saying that these times should be up for discussion after each election cycle.